### PR TITLE
AEROGEAR-2968 authentication page update

### PR DESCRIPTION
**Pull request by StephenCoady**
## Motivation

JIRA: https://issues.jboss.org/browse/AEROGEAR-2968

## Progress

- [x] Move button up the screen
- [x] Change colour of button
- [x] Change background image
- [x] Change button text colour

## Screenshot
https://docs.google.com/presentation/d/1FZL-Wy5D_wUo5ADrfhpRBh-Ny2rGauI3_C8SBrlk6rw/edit#slide=id.g3c52d834c7_0_177

screenshot 2018-06-19 21 06 39

### CODE REVIEW

**Comment by danielpassos** [comment]
Hey @StephenCoady, can you add a screenshot of the screen on the PR description?

**Comment by danielpassos** [comment]
Can you rename the background file for something like _authentication_background_, _identity_manager_background_?

File: app/src/main/res/layout/fragment_authentication.xml [hunk 1]
```diff
@@ -1,50 +1,53 @@
...
         android:layout_height="wrap_content"
-        android:text="@string/keycloak_login"
-        android:background="@color/orange"
-        android:layout_marginBottom="@dimen/authentication_button_margin_bottom"
         android:layout_alignParentBottom="true"
-        android:layout_alignParentStart="true" />
+        android:layout_alignParentStart="true"
+        android:layout_marginBottom="@dimen/authentication_button_margin_bottom"
+        android:layout_marginLeft="20dp"
+        android:layout_marginRight="20dp"
```
Comment (danielpassos) [inline app/src/main/res/layout/fragment_authentication.xml]: All dimensions should be moved to the res/values/dimens.xml file

**Comment by StephenCoady** [comment]
@danielpassos @wei-lee @secondsun mind taking a look?

**Comment by wei-lee** [comment]
tried it on 2 different size of emulators and they all look fine.

**Comment by danielpassos** [comment]
@StephenCoady Thanks for the changes

**Pull request merged**